include/rawcan_comm.hpp

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#include <linux/can.h>
+#include "i_comm.hpp"
+
+namespace roboteq { 
+
+    class rawcan_comm : public i_comm {
+
+        public:
+        
+        explicit rawcan_comm(canid_t _roboteq_can_id, std::string _ifname);
+        ~rawcan_comm(){}
+
+        // set methods
+        void SetParameters(int16_t _velocity, long _accel, long _decel, uint8_t _channel);
```

Comment:
![image](https://user-images.githubusercontent.com/56097508/69999262-3af07880-1526-11ea-9b9c-842ef315db68.png)

Reply:
I removed them to clear this error

Reply:
that error wasn't because of the overrides, it was because you typed override twice on the same function declaration(ReadAnalogInputs). you should add them all back.